code/game/area/Space Station 13 areas.dm

```diff
@@ -1443,9 +1443,13 @@ area/space/atmosalert()
 	icon_state = "quartoffice"
 
 /area/quartermaster/storage
-	name = "\improper Cargo Bay"
+	name = "\improper Cargo Warehouse"
 	icon_state = "quartstorage"

```

Comment:
I seperated the warehouse and cargo bay into two seperate areas. Was quite surprised to find they weren't seperate

This also involved making a new area icon for cargo bay, mapping in an APC for the warehouse, and remodelling the warehouse a little